docs/parkiNarodowe.js

```diff
@@ -0,0 +1,177 @@
+    new Region("Tatrzański Park Narodowy", "139;0;139"),
+    new Region("Wielkopolski Park Narodowy", "0;128;128"),
+    new Region("Wigierski Park Narodowy", "255;215;0"),
+    new Region("Woliński Park Narodowy", "65;105;225"),
+    new Region("Babiogórski Park Narodowy", "255;0;0"),
+    new Region("Drawieński Park Narodowy", "255;140;0")
+];
+
+function getLocalStgOr(key, default_value) {
+    x = localStorage.getItem(key);
+    if (x === null) return default_value;
+    else return Number(x);
+}
+
+function saveXPAndLvl(xp, lvl) {
+    localStorage.setItem("xp", xp);
+    localStorage.setItem("lvl", lvl);
+}
+
+window.onload = function () {
+    let experience = getLocalStgOr("xp", 0);
+    let lvl = getLocalStgOr("lvl", 1);
+    let curFibonachi = 2 + lvl;
+    let expCap = fibonachiNumber(curFibonachi);
+
+    const lvlBar = document.querySelector("#myBar");
+    const lvlText = document.querySelector("#overlayText");
+
+    function fibonachiNumber(point) {
+        if (point == 0) {
+            return 0;
+        }
+        else if (point == 2 || point == 1) {
+            return 1;
+        }
+        else {
+            let loop = 1;
+            let a = 1;
+            let b = 1;
+            while (loop < point) {
+                if (loop % 2 != 0) {
+                    a = a + b;
+                }
+                else {
+                    b = a + b;
+                }
+                loop += 1;
+            }
+            if (a > b) {
+                return a;
+            }
+            return b;
+        }
+    }
+
+    function moveBar(curAmmount, maxAmmount) {
+        var width = Math.ceil(curAmmount / maxAmmount * 100);
+        lvlBar.style.width = width + "%";
+    }
+
+    moveBar(experience, expCap);
+    lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    function expUp() {
+        experience += 1;
+        updateXP();
+    }
+    function updateXP() {
+        if (experience >= expCap) {
+            lvl += 1;
+            curFibonachi += 1;
+            expCap = fibonachiNumber(curFibonachi);
+            experience = 0;
+            if (expCap <= 0) {
+                console.log("Wow. Int overload... that was so unexpected... If your testing this library, then I understand how you got here, else... go touch some grass, pls");
+            }
+        }
+
+        saveXPAndLvl(experience, lvl);
+
+        moveBar(experience, expCap);
+        lvlText.innerText = experience + "/" + expCap + " XP, level " + lvl;
+    }
+
+    // Get the HTML Canvas
+    const canvas = document.querySelector("#image_canvas");
+    // Get the canvas context
+    const ctx = canvas.getContext('2d');
+
+    const name = document.querySelector('#name');
+    const points = document.querySelector('#points');
+
+    const chosen = document.querySelector('#chosen');
+
+    // Set this variable to any image source
+    const IMAGE_SRC = 'parkiNarodoweRaw.png';
```

Comment:
I may be wrong but extracting some things into an init function may no be such a bad idea